src/main/java/frc/utils/time/TimeUtils.java

```diff
@@ -0,0 +1,50 @@
+package frc.utils.time;
+
+import edu.wpi.first.hal.HALUtil;
+import frc.utils.Conversions;
+import frc.utils.alerts.Alert;
+import frc.utils.alerts.AlertManager;
+import frc.utils.alerts.PeriodicAlert;
+import org.littletonrobotics.junction.Logger;
+
+public class TimeUtils {
+
+	public static final double DEFAULT_CYCLE_TIME_SECONDS = 0.02;
```

Comment:
shouldn't this be in the constants file?

Reply:
why? its better to keep in the usage classs constants that meant for outside usage

Reply:
What's the difference between this constant and the ones in the constants file?

Reply:
in the const file its constants that meant for time utils usage, in here its constants that meant to use by other classes in code that relate to time. like default voltage in battery utils